packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +545,97 @@ void main() {
+          ];
+          final SingleChildScrollView scrollView = SingleChildScrollView(
+            child: SizedBox(
+              height: 100,
+              width: 100,
+              child: ReorderableListView(
+                key: const Key('list'),
+                physics: const AlwaysScrollableScrollPhysics(),
+                onReorder: (int oldIndex, int newIndex) {},
+                children: widgetList,
+                scrollController: ScrollController(),
+              )
+            ),
+          );
+          await tester.pumpWidget(MaterialApp(
+          home: SizedBox(
+            height: 100,
+            width: 100,
+            child: scrollView,
+          ),
+        ));
+
+          final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await gesture.moveBy(_kGestureOffset);
+          // Move back to original position.
+          await gesture.moveBy(Offset(-_kGestureOffset.dx, -_kGestureOffset.dy));
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          final TestGesture overscrollGesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await overscrollGesture.moveBy(_kGestureOffset);
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          expect(tester.takeException(), isNull);
+        });
+
+        testWidgets('Default ReorderableListViews are not always scrollable', (WidgetTester tester) async {
+          final ReorderableListView view = ReorderableListView(
+            scrollDirection: Axis.horizontal,
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          );
+          expect(view.physics, isNot(isInstanceOf<AlwaysScrollableScrollPhysics>()));
+        });
+
+        testWidgets('physics:AlwaysScrollableScrollPhysics overrides default behavior', (WidgetTester tester) async {
+          bool scrolled = false;
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Directionality(
+              textDirection: TextDirection.ltr,
+              child: NotificationListener<OverscrollNotification>(
+                onNotification: (OverscrollNotification message) { scrolled = true; return false; },
+                child: ReorderableListView(
+                  physics: const AlwaysScrollableScrollPhysics(),
+                  children: const <Widget>[],
+                  onReorder: (int oldIndex, int newIndex) {},
+                ),
+              ),
+            ),
+            )
+          );
+          await tester.dragFrom(const Offset(100.0, 100.0), const Offset(0.0, 100.0));
+          expect(scrolled, isTrue);
+        });
+
+        testWidgets('physics:ScrollPhysics behaves as expected in ReorderableListView', (WidgetTester tester) async {
```

Comment:
What does `as expected` mean here? Is is that there is over-scroll is not being used? If so, maybe changed the `scrolled` property name to `wasOverscrolled`  or something similar and update the unit test name to suggest that.

Reply:
Yes, the test is checking to see if there is over-scrolling happening. I updated the property name.